### ISSUE TITLE
Loss of Convergence in Iterative Loop (iter.00) Despite Initial Convergence Rates

### ISSUE THREAD

**Issue by Shangguanying992** [deepmodeling/deepks-kit#70]
While using DeepKS, I observed an anomaly regarding convergence rates during the initial iteration (iter. init) and subsequent iterative loop (iter.00).

During the first iteration (iter. init), the convergence rates were satisfactory with a training set rate of 0.77 and a testing set rate of 0.88. 
However, upon entering the iterative loop (iter.00), the convergence rates dropped to 0, rendering further computation impossible.
Another user reported this issue as well.
![屏幕截图 2024-03-06 145959](https://github.com/deepmodeling/deepks-kit/assets/147317022/ecd1defd-8222-42ed-9410-864ea0510300)

Additional Context:
Software: DeepKS, abacus 3.5.3
The DeePKS water_single example can run correctly on my machine. 

Thank you so much for your help in addressing this issue.

**Comment by Shangguanying992** [comment]
[example.zip](https://github.com/deepmodeling/deepks-kit/files/14521491/example.zip)

**Comment by y1xiaoc** [comment]
Thanks for the question. This happens occationally when the first iteration is trained so "hard" that it overfits the data. I would suggest reduce the number of training epoch for first iteration.

**Comment by Shangguanying992** [comment]
I tried changing the value of n_epoch from 500 to 10, 100, but it seems that the situation does not change. Are there any other solutions?

**Comment by y1xiaoc** [comment]
To clarify, you may want to reduce the training length for the _init_ train (`iter.init`), which is controlled by paramteres in `init_train`, like [here](https://github.com/deepmodeling/deepks-kit/blob/7978b398a5edd688e3fffc3c29f839d3c7b99602/examples/water_cluster/args.yaml#L188) in the example. I think the one you are changing is for the following iterations.

**Comment by Shangguanying992** [comment]
I changed it in params.yaml, like the file below. Do I truly understand? Or do I need to add a file named args.yaml?
[params.yaml.zip](https://github.com/deepmodeling/deepks-kit/files/14568840/params.yaml.zip)

**Comment by y1xiaoc** [comment]
You may need to use the args.yaml. You can check the iter.init folder to see if the training has been redone with reduced epochs.

**Comment by Shangguanying992** [comment]
I‘m sure that the training has been redone with a reduced epoch, but the convergence rate is still 0. In this case, is there a recommended n_epoch value?
![屏幕截图 2024-03-15 162701](https://github.com/deepmodeling/deepks-kit/assets/147317022/1bbb9905-f47c-428b-b5eb-15dbcd22a69b)
![屏幕截图 2024-03-15 164426](https://github.com/deepmodeling/deepks-kit/assets/147317022/66536e4a-ee55-4db4-a1f6-dc44cdd4ed55)

**Comment by ouqi0711** [comment]
You may try symmetrizing the descriptors via modifying the init_train block in params.yaml as suggested [here](https://deepks-kit.readthedocs.io/en/latest/inputs-preperation.html#params-yaml). (See last part of params.yaml)

**Comment by Shangguanying992** [comment]
I have tried to modify the params.yaml as suggested, but it still has the same problem. I changed  n_epoch and start_lr.

**Comment by xuwenli-nku** [comment]
> 在使用 DeepKS 时，我观察到初始迭代 （iter. init） 和随后的迭代循环 （iter.00） 期间收敛率出现异常。
> 
> 在第一次迭代 （iter. init） 期间，收敛率令人满意，训练集率为 0.77，测试集率为 0.88。然而，在进入迭代循环 （iter.00） 时，收敛率下降到 0，无法进一步计算。另一位用户也报告了此问题。 ![屏幕截图 2024-03-06 145959](https://github.com/deepmodeling/deepks-kit/assets/147317022/ecd1defd-8222-42ed-9410-864ea0510300)
> 
> 附加上下文： 软件：DeepKS、abacus 3.5.3 DeePKS water_single示例可以在我的机器上正常运行。
> 
> 非常感谢您帮助解决此问题。

您好，我是一名刚开始学习deepks的初学者，想向您请教如何才能复现官网中给出的water_single和water_cluster的例子呢，我在复现的时候err文件中给出了pyscf版本与目前deepks库不匹配的问题，还有就是这个deepks的任务可以通过提交到远程服务器slurm上跑通嘛，因为我被困扰了很久，实在找不到解决方法，希望您能给我一点建议，麻烦了！
